sections/shared/Layout/AppLayout/AppLayout.tsx

```diff
@@ -1,32 +1,80 @@
 import { FC, memo } from 'react';
+import styled from 'styled-components';
 
 import { DesktopOnlyView, MobileOrTabletView } from 'components/Media';
+import { Body } from 'components/Text';
 import NotificationContainer from 'constants/NotificationContainer';
-import { FullScreenContainer, MobileScreenContainer } from 'styles/common';
+import { MobileScreenContainer } from 'styles/common';
 
+import GitHashID from './GitHashID';
 import Header from './Header';
 import MobileUserMenu from './Header/MobileUserMenu';
 
 type AppLayoutProps = {
 	children: React.ReactNode;
 };
 
+const Footer = () => {
+	return (
+		<FooterContainer>
+			<Body color="secondary">Fully operational</Body>
+			<GitHashID />
```

Comment:
This isn't quite centred, maybe we can have fixed widths at the edges or equal flex's?